typescript/cli/src/deploy/warp.ts

```diff
@@ -329,8 +329,10 @@ function writeWarpUiTokenConfig(
       'No base Hyperlane token address deployed and no foreign deployment specified',
     );
   }
+  const chain = multiProvider.getChainMetadata(origin);
+  if (chain.protocol !== ProtocolType.Ethereum) throw Error('Unsupported VM');
   const commonFields = {
-    chainId: multiProvider.getChainId(origin),
+    chainId: multiProvider.getDomainId(origin),
```

Comment:
I think this should stay as `getChainId`. You can use `getChainIdNumber` if the types aren't happy